src/current/_includes/releases/cloud/2023-05-15.md

```diff
@@ -11,7 +11,7 @@ In addition to many of the Feature Highlights in the [CockroachDB v23.1.0 Releas
     - The cluster’s regional configuration applies to all databases by default, so it is not necessary to run [`ALTER DATABASE ... ADD REGION`](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/alter-database#add-region) to configure regions when adding a database to the cluster.
     - [`SURVIVE REGION FAILURE`](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/alter-database#survive-zone-region-failure) is supported to achieve [region-level survival goals](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/multiregion-overview#survival-goals).
 
-- [Creating CockroachDB {{ site.data.products.dedicated }} clusters on Azure](https://www.cockroachlabs.com/docs/cockroachcloud/cockroachdb-dedicated-on-azure) is now available in [limited access](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/cockroachdb-feature-availability). To enroll your organization and begin testing this feature, contact your Cockroach Labs account team.
+- [Creating CockroachDB {{ site.data.products.dedicated }} clusters on Azure](https://www.cockroachlabs.com/docs/cockroachcloud/cockroachdb-advanced-on-azure) is now available in [limited access](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/cockroachdb-feature-availability). To enroll your organization and begin testing this feature, contact your Cockroach Labs account team.
```

Comment:
It's an old RN so not likely to be read or relied on by many, but to avoid potential confusion, one option is to switch this to the `advanced` var, and add a sentence or sub-bullet: "This update originally referred to the product name at the time of the release, CockroachDB Dedicated." Same could work for all the other similar mentions.

I thought about doing it the other way around, too - leaving it alone and adding a caveat - but that seemed more clunky; I couldn't quite get it right.

Reply:
As discussed on Slack: rephrase as `(named Dedicated at the time of this release)`